include/swift/Basic/LangOptions.h

```diff
@@ -43,6 +43,8 @@ namespace swift {
     Endianness,
     /// Runtime support (_ObjC or _Native)
     Runtime,
+    /// Conditional import
+    CanImport,
   };
   enum { NumPlatformConditionKind = 4 };
```

Comment:
Could you add comments about why `NumPlatformConditionKind` does not count `CanImport`?
Or increment this, and update `PlatformConditionValues` declaration with `NumPlatformConditionKind - 1` (with comment)

Reply:
Is having `NumPlatformConditionKind` defined in the first place really necessary?

Reply:
Not necessary, but I added this in https://github.com/apple/swift/commit/1f3c6622#diff-f544d77b7be49d30361f0f41d9f38244L275 because people forget to update this value.

Reply:
My thinking is that even if people update this value, it's better to provide recommended bounds in powers of two.